src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_Basics/Task4_1.java

```diff
@@ -0,0 +1,19 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_Basics;
+
+import java.util.Scanner;
+
+public class Task4_1 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int sum = 0;
+        System.out.println("Enter a last number please:");
+        int secondNUmber = scanner.nextInt();
+        System.out.println("The sum for our numbers is:");
```

Comment:
In Task4 the final sout is written in a single line, could you please also make it similar here?

Reply:
Done.